provider-gui/services/contract/contractOverview.ts

```diff
@@ -0,0 +1,98 @@
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api'
+import { GovernanceStatus } from '@prosopo/captcha-contract'
+import { ProcaptchaClientConfigOutput } from '@prosopo/types'
+import { ProsopoCaptchaContract } from '@prosopo/contract'
+import { ContractAbi as abiJson } from '@prosopo/captcha-contract'
+import { hexToString } from '@polkadot/util'
+
+const deployedContracts = {
+    rococo: {
+        wsProviderUrl: 'wss://rococo-contracts-rpc.polkadot.io:443',
+        contracts: ['5HiVWQhJrysNcFNEWf2crArKht16zrhro3FcekVWocyQjx5u'],
+    },
+    development: { wsProviderUrl: 'ws://localhost:9944', contracts: [] },
```

Comment:
likewise with networks, would be good to be able to add our own somewhat easily so I can reuse this tool on astar, etc, without having to recompile

Reply:
Ticket in